src/index.html

```diff
@@ -11,6 +11,46 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      class="form"
+      action="/"
+      method="get"
+      data-qa="big">
+      <label class="lable">
+        <img
+          class="img__form"
+          src="images/Search.svg"
+          alt="search__icon"
+         >
```

Comment:
The open and close tags should be on the same level
```suggestion
        <img
          class="img__form"
          src="images/Search.svg"
          alt="search__icon"
        >
```